Sources/Core/Transaction/Policies.swift

```diff
@@ -54,12 +54,6 @@ public struct Policies {
         }
 
     public static var auto: Policies {
-        return Policies(
-            noncePolicy: .latest,
-            gasLimitPolicy: .automatic,
-            gasPricePolicy: .automatic,
-            maxFeePerGasPolicy: .automatic,
-            maxPriorityFeePerGasPolicy: .automatic
-        )
+        Policies()
```

Comment:
minor: could be written in one line now `public static var auto = Policies()`

Reply:
I've updated this to follow: `public private(set) static var auto = Policies()`